gammapy/datasets/flux_points.py

```diff
@@ -146,6 +141,32 @@ def write(self, filename, overwrite=True, **kwargs):
         table["mask_safe"] = self.mask_safe
         table.write(make_path(filename), overwrite=overwrite, **kwargs)
 
+    @classmethod
+    def read(cls, filename, name=None, format="gadf-sed"):
+        """Read pre-computed flux points and create a dataset
+
+        Parameters
+        ----------
+        filename : str
+            Filename to read from.
+        name : str
+            Name of the new dataset.
+        format : {"gadf-sed"}
+            Format of the dataset file.
+
+        Returns
+        -------
+        dataset : `FluxPointsDataset`
+            FluxPointsDataset
+        """
+        from gammapy.estimators import FluxPoints
+
+        filename = make_path(filename)
+        table = Table.read(filename)
```

Comment:
Is there any specific reason to go though `Table.read()` and not `FluxPoints.read()`?

Reply:
`.write` is writing a `Table`, so I did similarly here. Would `FluxPoints.read()` work in all cases?